### PR TITLE
Setup pkgdown and travis

### PR DESCRIPTION
**Pull request by peterdesmet**
Also included:

- Make @florisvdh maintainer of package in DESCRIPTION
- Add code of conduct
- Add contributing guide based on https://gist.github.com/peterdesmet/e90a1b0dc17af6c12daf6e8b2f044e7c

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,25 +1,24 @@
...
-           email = "first.last@example.com",
-           comment = c(ORCID = "YOUR-ORCID-ID"))
 Description: The R-package watina contains functions to query
     and process data from the Watina database at the Research Institute for
     Nature and Forest (INBO). This database provides
     groundwater level and chemical data, mainly from natural 
     areas in Flanders (Belgium).
+Authors@R: c(
+    person("Floris", "Vanderhaeghe", role = c("aut", "cre"), email = "floris.vanderhaeghe@inbo.be", comment = c(ORCID = "0000-0002-6378-6229")),
+    person("Research Institute for Nature and Forest (INBO)", role = "cph", email = "info@inbo.be")
```
Comment (florisvdh) [inline DESCRIPTION]: Right, I still forgot INBO; thanks!
I don't know where I've got it from (probably another INBO package), but in the n2khab package I use the role `c("cph", "fnd")` for INBO (funder). What is your view on this?

**Comment by florisvdh** [comment]
Was still reviewing ;-)

A really nice contribution, thanks a lot!! Very beautiful contributing guide!

File: DESCRIPTION (hunk 1 above)
Comment (peterdesmet) [reply DESCRIPTION]: I would keep it to copyright holder. Note that the contributing guide says we are not directly funded to maintain this pkg. :smile:

**Comment by peterdesmet** [comment]
Sorry for quick merge, was done to setup website. If you have further comments, let me know.

**Comment by florisvdh** [comment]
I understand. In `README.md`, I'll add a clarification at the beginning the contribution / git workflow described therein. That workflow is intended for those who directly cooperate on the repo (INBO-coworkers in this case, as the watina-db is not yet public). Other people can be directed to `CONTRIBUTING.md`.

File: DESCRIPTION (hunk 1 above)
Comment (florisvdh) [reply DESCRIPTION]: OK, I'll correct it over there.

**Comment by peterdesmet** [comment]
Is online: https://inbo.github.io/watina/

**Comment by florisvdh** [comment]
Really great, thanks!

**Pull request merged**